File: snippets/snippets/python/embed-jobs-post.py
import cohere

co = cohere.Client()

# start an embed job
job = co.embed_jobs.create(
    dataset_id="my-dataset-id", input_type="search_document", model="embed-english-v3.0"
)

# poll the server until the job is complete
response = co.wait(job)

print(response)

File: snippets/snippets/node/embed-jobs-post.js
import { CohereClient } from 'cohere-ai';

const cohere = new CohereClient({});

(async () => {
  const embedJob = await cohere.embedJobs.create({
    datasetId: 'my-dataset',
    inputType: 'search_document',
    model: 'embed-v4.0',
  });

  console.log(embedJob);
})();
